dir.common.props

```diff
@@ -0,0 +1,35 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
If this is being included in SDK project types, should we have a property for the `TargetFramework`. Then we can define it once for all test projects?

Reply:
Put this in a common props file - see my comment below.